src/components/fabSpeedDial/fabSpeedDial.js

```diff
@@ -182,6 +187,12 @@
       var ctrl = element.controller('mdFabSpeedDial');
       var items = el.querySelectorAll('.md-fab-action-item');
 
+      // Grab our element which stores CSS variables
```

Comment:
Funny thing, I actually meant variables there. The SCSS defines the z-index as a variable and I wanted to pass that along to the JS code in a clean way. This ensured that I had a known element to look for those variables.

I'm happy to rename/refactor it though if you think something else makes more sense :-)